### PR TITLE
Add release data tables

### PR DESCRIPTION
**Pull request by chandankumar4**
Signed-off-by: chandan kumar <chandan.kumar@mayadata.io>
- Api added for release branch pipeline data are: 
      /api/release/openshift
- Total api available
     /api/master/openshift
     /api/master/build
     /api/release/openshift

### CODE REVIEW

**Comment by chandankumar4** [comment]
@bhaskarhc Api are changed, change in the dashboard accordingly

File: database/router/router.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package router
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/golang/glog"
+	"github.com/openebs/ci-e2e-status/handler"
```
Comment (bhaskarhc) [inline database/router/router.go]: replace URL `github.com/mayadata-io/ci-e2e-status/handler`

File: handler/build.go [hunk 2]
```diff
@@ -12,12 +13,12 @@ import (
 	"github.com/openebs/ci-e2e-status/database"
```
Comment (bhaskarhc) [inline handler/build.go]: replace URL `github.com/mayadata-io/ci-e2e-status/database`

File: handler/openshift.go [hunk 3]
```diff
@@ -11,8 +11,8 @@ import (
 	"github.com/openebs/ci-e2e-status/database"
```
Comment (bhaskarhc) [inline handler/openshift.go]: replace URL `github.com/mayadata-io/ci-e2e-status/database`

File: main.go [hunk 4]
```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/golang/glog"
 	_ "github.com/lib/pq"
 	"github.com/openebs/ci-e2e-status/database"
+	"github.com/openebs/ci-e2e-status/database/router"
 	"github.com/openebs/ci-e2e-status/handler"
```
Comment (bhaskarhc) [inline main.go]: replace URL 

- `github.com/mayadata-io/ci-e2e-status/database`
- `github.com/mayadata-io/ci-e2e-status/database/router`
- `github.com/mayadata-io/ci-e2e-status/handler`

**Review by bhaskarhc — APPROVED** [review]
LGTM

**Pull request merged**